backend/main.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+const GenericController = require('./GenericController')
+const port = 4000
+
+function get_models() {
+    const Sequelize = require("sequelize");
+    const initModels = require("./models/init-models").initModels;
+    var sequelize = new Sequelize("northwind", "root", "root", { dialect: 'mysql' });
+    var models = initModels(sequelize);
+    return models
+}
+
+let models = get_models()
+let customers = models.customers
+let employees = models.employees
+let invoices = models.invoices
+let orders = models.orders
+let products = models.products
+let suppliers = models.suppliers
```

Comment:
Poti scrie asa:
```suggestion
const { customers, employees, invoices, orders, products, suppliers } = get_models()
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring

Reply:
Done.